deployment/GI03FLMB_Deploy.csv

```diff
@@ -1,118 +1,118 @@
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH100,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-11600,59.71427,-39.31329,,2729,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH400,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-11599,59.71427,-39.31329,,2429,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH700,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-10267,59.71427,-39.31329,,2129,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11669,59.71427,-39.31329,,1829,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB100,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-12195,59.71427,-39.31329,,2729,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB400,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11967,59.71427,-39.31329,,2429,2829,
-AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB700,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11964,59.71427,-39.31329,,2129,2829,
-AR30-03,,AR35-05,,GI03FLMB-RIM01-00-SIOENG000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGCON-SIOCON-14885,59.71427,-39.31329,,1507,2829,
-AR30-03,,AR35-05,,GI03FLMB-RIM01-02-ADCPSL007,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-ADCPSL-18351,59.71427,-39.31329,,500,2829,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-CTDMOH000,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-CTDMOH-12224,59.7128,-39.3204,,1825,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-CTDMOH100,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-CTDMOH-12226,59.7128,-39.3204,,2725,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-CTDMOH400,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-CTDMOH-12219,59.7128,-39.3204,,2425,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-CTDMOH700,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-CTDMOH-12218,59.7128,-39.3204,,2125,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-VELPTB000,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-VELPTB-11973,59.7128,-39.3204,,1825,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-VELPTB100,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-VELPTB-11981,59.7128,-39.3204,,2725,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-VELPTB400,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-VELPTB-11976,59.7128,-39.3204,,2425,2825,
+KN221-04,,AT30-01,,GI03FLMB-RI000-00-VELPTB700,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGINS-VELPTB-11974,59.7128,-39.3204,,2125,2825,
+KN221-04,,AT30-01,,GI03FLMB-RIM01-00-SIOENG000,1,1,2014-09-16T13:24:00,2015-08-20T12:56:00,CGMGI-03FLMB-00001,,CGCON-SIOCON-13285,59.7128,-39.3204,,1501,2825,

```

Comment:
Controller depth should be 1508 (per the cruise appendix)

Reply:
discussed w/ SW and will change Main Controller depth to what it originally was at 1508 m.